modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/info/item/provider/FileEntryInfoItemFieldValuesProvider.java

```diff
@@ -276,6 +276,16 @@ private String _getDisplayPageURL(
 						FileEntryInfoItemFields.downloadURL, downloadURL));
 			}
 
+			String previewURL = _dlURLHelper.getPreviewURL(
+				fileEntry, fileEntry.getFileVersion(), themeDisplay,
+				StringPool.BLANK, false, true);
+
+			if (Validator.isNotNull(downloadURL)) {
```

Comment:
Shouldn't this be `Validator.isNotNull(previewURL)`?

Reply:
That is totally right :)

Reply:
Fixed and pushed.